content/blog/getting-started-with-k8s-part3/index.md

```diff
@@ -365,3 +366,9 @@ Using a Helm chart to deploy an application is the simplest solution if it’s a
 ## Conclusion
 
 This article demonstrated three ways to deploy an application in Kubernetes. In the next installment, we’ll examine how to maintain your application with [configmaps](https://kubernetes.io/docs/concepts/configuration/configmap/), perform rollouts and different deployment strategies, adopting existing resources, and more.
+
+This is the second in a series of articles on using infrastructure as code for Kubernetes. In the next article, we'll examine a complex application deployment and a deployment using a Helm chart. Until the next post, learn more about Kubernetes with these resources.
```

Comment:
This refs the 2nd part of the series, but the PR is for part 3

Reply:
Odd, local repo has the changes. Pushing again.